### ISSUE TITLE
About Label Shuffle

### ISSUE THREAD

**Issue by zhangchbin** [hongjoon0805/SS-IL-Official#7]
what does "label shuffled" mean? Thanks.

**Comment by hongjoon0805** [comment]
To permute the order of classes, we shuffled the class array. The "label shuffled" is the permutation of np.arange(classes).

**Comment by zhangchbin** [comment]
Does it mean the class order is random? If I run this code multi-times, the class order for each running is not the same, right?

**Comment by jihwankwak** [comment]
Yes :) the class order is randomly shuffled depending on the random seed.
